indra/newview/llfloatersnapshot.cpp

```diff
@@ -51,7 +51,10 @@ LLSnapshotFloaterView* gSnapshotFloaterView = NULL;
 const F32 AUTO_SNAPSHOT_TIME_DELAY = 1.f;
 
 const S32 MAX_POSTCARD_DATASIZE = 1572864; // 1.5 megabyte, similar to simulator limit
-const S32 MAX_TEXTURE_SIZE = 512 ; //max upload texture size 512 * 512
+// <FS:Chanayane> 2048x2048 snapshots upload to inventory
+//const S32 MAX_TEXTURE_SIZE = 512 ; //max upload texture size 512 * 512
```

Comment:
There's no need to save the old versions as comments, please remove them as well.